src/Mod/CAM/Path/Post/UtilsArguments.py

```diff
@@ -88,10 +88,13 @@ def init_argument_defaults(argument_defaults: Dict[str, bool]) -> None:
 
 def init_arguments_visible(arguments_visible: Dict[str, bool]) -> None:
     """Initialize the flags for which arguments are visible in the arguments tooltip."""
-    arguments_visible["bcnc"] = False
     arguments_visible["axis-modal"] = True
     arguments_visible["axis-precision"] = True
+    arguments_visible["bcnc"] = False
```

Comment:
Most of the arguments are self-evident.  I know 'bcnc' isn't a new arg but this might be a good opportunity to clarify what it does.

Reply:
Neither do I :-)

It was in the code that I started from, but otherwise undocumented.